build-system/tasks/helpers.js

```diff
@@ -456,6 +456,7 @@ async function esbuildCompile(srcDir, srcFilename, destDir, options) {
         incremental: !!options.watch,
         logLevel: 'silent',
         external: options.externalDependencies,
+        mainFields: ['module', 'browser', 'main'],
```

Comment:
Let's update the test running with this, too.

Reply:
What test?

Reply:
All of them: https://github.com/ampproject/amphtml/blob/31cf6fd29a797479d9fd22d5eb2fcdd96960797a/build-system/tasks/runtime-test/runtime-test-base.js#L232-L240

Reply:
Ah, forgot about this. Sure